cf-usb/tasks/mysql-build.sh

```diff
@@ -2,8 +2,9 @@
 set -o nounset -o errexit -o xtrace
 export GOPATH="${PWD}"
 export START_DIR="${PWD}"
+service=mysql
```

Comment:
Perhaps this can be a [run arg](http://concourse.ci/running-tasks.html#task-run-args) or [environment variable](http://concourse.ci/running-tasks.html#task-params) from the pipeline, so we only have one script for all sidecars?

Reply:
Yes, this chunk was preparation for things like that, without knowing how to specify such.
Thanks to the links I can do that.

Reply:
Would we still need separate task yml files ?

Reply:
Looks to be. Done.

Reply:
Done with commit 345074c906484e2478bf6d1289bd535795c298ff, in testing.